external/CMakeLists.txt

```diff
@@ -120,6 +120,7 @@ ExternalProject_Add(libwally-core
                   ${CMAKE_CURRENT_SOURCE_DIR}/libwally-core/configure
                   ${CONFIGURE_FLAGS}
                   ${LIBWALLY_CONFIGURE_FLAGS}
+  COMMAND         find ${CMAKE_CURRENT_BINARY_DIR}/libwally-core -name Makefile | xargs sed -i "s/-O3/-Os/"
```

Comment:
This is very not nice :D Is there really no way to configure it or pass it as an env var or sth? 

otherwise you can also make a new branch here: https://github.com/shiftdevices/libwally-core